src/components/UI/CardContentDetailMock.vue

```diff
@@ -33,42 +31,46 @@ export default defineComponent({
 
 <style lang="scss" module>
 .container {
-  border-bottom: solid 1.5px #d9d9d9;
   text-align: left;
-  padding-top: 1rem;
-  padding-bottom: 1rem;
-}
-.container:first-of-type {
-  padding-top: 0;
-}
-.container:last-of-type {
-  padding-bottom: 0;
-  border: none;
+  padding: 1rem;
+  border-bottom: solid 1px $border;
+  &:last-child {
+    border: none;
+  }
 }
 .tableItemTitle {
   min-width: 10rem;
   text-align: left;
 }
-.tableItemDate {
-  min-width: 8rem;
-  padding-left: 0.8rem;
+.column {
+  margin: 0.5rem 0;
+  display: grid;
+  grid-template-columns: 50%;
+}
+
+.resTimeLimit {
+  grid-row: 1/2;
+  grid-column: 1/2;
+  text-align: left;
+}
+.modifiedAt {
+  grid-row: 1/2;
+  grid-column: 2/3;
+  text-align: left;
 }
 .boxTitle {
-  border-radius: 4px;
-  margin-bottom: 0.8rem;
-  min-width: 320px;
-  height: 20px;
+  border-radius: 1rem;
+  min-width: 20rem;
```

Comment:
```suggestion
  width: min(20rem, 100%);
```
画面幅が小さいとはみ出るのでこんな感じにしてほしいです
↓も必要です
```
.tableItemTitle {
  width: 100%;
  display: block;
  text-align: left;
}
```